test/katello_test_helper.rb

```diff
@@ -86,24 +86,23 @@ module FixtureTestCase
     Dir.mkdir(self.fixture_path)
     FileUtils.cp(Dir.glob("#{Katello::Engine.root}/test/fixtures/models/*"), self.fixture_path)
     FileUtils.cp(Dir.glob("#{Rails.root}/test/fixtures/*"), self.fixture_path)
+
+    taxonomies_file = "#{self.fixture_path}/taxonomies.yml"
+    taxonomies = YAML.safe_load(File.read(taxonomies_file))
+    taxonomies.values.each do |taxonomy|
+      next unless taxonomy['type'] == 'Organization'
+      taxonomy['label'] = taxonomy['name'].tr(' ', '_')
```

Comment:
I'm hoping that this will fix the problem. My suspicion is that *something* was resetting the orgs in the database with fixture data. Adding the org label to the fixture directly should solve the issue. I considered keeping our own copy of taxonomies.yml with the labels built-in but opted to let Foreman be the authority on those